src/lcm_republisher.cpp

```diff
@@ -165,6 +169,16 @@ class LCMRepublisher
       rgb_height = rgb_msg->height;
       depth_width = depth_msg->width;
       depth_height = depth_msg->height;
+
+      if (rgb_width != depth_width || rgb_height != depth_height) {
+        ROS_WARN("Expected rgb dimensions (%d,%d) to be equal to depth dimensions (%d,%d)",
+        rgb_width, rgb_height, depth_width, depth_height);
+      }
+
+      //$ user-specified outgoing image dimensions for both depth and rgb
```

Comment:
Please don't add dollar signs at the start of comment, that's meant to indicate I wrote it haha

Reply:
Thought it was some special auto doc spec I had to adhere to :)